.palantir/revapi.yml

```diff
@@ -3,6 +3,14 @@ versionOverrides:
 acceptedBreaks:
   apache-iceberg-0.14.0:
     org.apache.iceberg:iceberg-api:
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.PartitionField"
+      new: "class org.apache.iceberg.PartitionField"
+      justification: "{Introduce schema id mapping for partition spec field}"
+    - code: "java.class.defaultSerializationChanged"
+      old: "class org.apache.iceberg.PartitionKey"
+      new: "class org.apache.iceberg.PartitionKey"
```

Comment:
auto moved by rev-api or spotless.

Reply:
it's auto-moved by revapi if you execute the gradle cmd it proposes, as spotless doesn't touch this file